### PR TITLE
Adding default email template

### PR DESCRIPTION
**Pull request by philipbed**
Adding a generic email template in the case we only know the officials and location, but don't have an available budget available. I wanted to get some 👀 on it so we have a consensus.

Got this from [here](https://defund12.org/huntsville)

### CODE REVIEW

**Comment by qxh5696** [comment]
Shouldn't `Huntsville PD` be replaced with `[INSERT CITY] PD`?

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
...
+# hitemup
+
+## Default email template
+(Don't forget to replace the [x]'s with your information!)
+**To:** [INSERT EMAILS]
+
+**Subject:** Defund [INSERT CITY] PD 
+
+**Message:**
+Hi, my name is [NAME] and I am a resident of [NEIGHBORHOOD]. I am asking to redirect money away from the Huntsville PD and into the following social services: [LIST - Community Development, COVID Relief, Education, Parks & Recreation]. I request that an emergency meeting be called before the fiscal year goes into effect in order to deny the mayor’s proposed budget and reallocate these funds to resources from which the citizens can benefit.
```
Comment (qxh5696) [inline README.md]: In this line here.

Comment (philipbed) [reply README.md]: Yes, good catch

Comment (qxh5696) [reply README.md]: Looks good to me 👍🏿

**Review by doshmajhan — APPROVED** [review]

**Pull request merged**